deluge/crypto_utils.py

```diff
@@ -105,9 +103,7 @@ def generate_ssl_keys():
     """
     This method generates a new SSL key/cert.
     """
-    from deluge.common import PY2
-
-    digest = 'sha256' if not PY2 else b'sha256'
+    digest = b'sha256'
```

Comment:
This doesn't seem correct, should be string not bytes. 

Discussion for reference: https://github.com/pyca/pyopenssl/issues/730

Reply:
Stupid docstring... The IDE warned me about it being bytes and not string because of it.
Will change in the next push.